packages/web-components/src/donut-chart/donut-chart.ts

```diff
@@ -0,0 +1,316 @@
+import { attr, FASTElement, nullableNumberConverter } from '@microsoft/fast-element';
+import { arc as d3Arc, pie as d3Pie } from 'd3-shape';
+import { createTabster, getMover, getTabsterAttribute, MoverDirections } from 'tabster';
```

Comment:
We need tabster to support accessibility for our controls. Do you have a suggestion for an alternative we could use instead of tabster. We are not aware of any other alternative.

Reply:
We need more functionalities from tabster like grouping elements, allowing circular navigation, navigation support using all 4 arrow keys provided by tabster. We don't want to rebuild tabster functionality on our own.